public/Website/index.html

```diff
@@ -0,0 +1,29 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/bootstrap.css"
+    <link rel="stylesheet" href="css/style.css"
+</head>
+<body>
+
+    <style>
+        body {
+          background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('background.jpg');
+        }
+        </style> 
```

Comment:
Needs to be on CSS file.  Not good practice to have style tag on HTML - look into separation of concerns

Reply:
for reference - look into quick start section https://getbootstrap.com/docs/5.0/getting-started/introduction/